requirements-vllm.txt

```diff
@@ -1,205 +1,205 @@
-aioboto3==13.3.0 ; python_version >= "3.11" and python_version < "3.12"
```

Comment:
Can you pl drop the dependency updates? Or make them in a separate PR if intentional.